recipes/fs.webdavfs/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "fs.webdavfs" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "dd326400ddd188f61f8314209162f9d0fe2753eb1a6c005407b583b92b9ea549" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```

Comment:
You'll have to add `nose` to the `build` requirements due to:

https://github.com/PyFilesystem/webdavfs/blob/master/setup.py#L45

Reply:
next try ;)

Reply:
this is one of the reasons I recognized pyfilesystem2 and currently refactor our package ;)

Reply:
Is it ok to have this name? or is it mandatory to have the origin project name?

Reply:
I think the current name is fine as the project is referred to `fs.webdavfs` on pypi.io and in the README on the github page. What alternative were you considering?

Reply:
I think the name is the best choice as it is based on fs and hooks into that. Also we have fs.sshfs too.

It is just because of the project name which is different, webdavfs